src/chapters/introduction/about-this-class.rst

```diff
@@ -0,0 +1,67 @@
+=========================
+About LaunchCode Programs
+=========================
+
+Goals
+=====
+
+The goal of our programs is to build our problem solving skills and to learn how to learn.
+Whether you use the coding skills you learn in this program to get a job as a developer is up to you.
+No matter the path you take after this program, learning how to learn will help you continually adapt to the changing needs of your industry.
+
+A goal for us is that we teach you skills to prepare you for jobs in technology.
+We are here to teach you technologies that are prevalent throughout a wide variety of industries.
+
+Blended learning
+================
+
+.. index:: ! blended learning model
+
+We only have a short amount of time in class to learn a lot, so using a **blended learning model** helps us make the most of our time in this class.
+A blended learning model incorporates in-class learning with online materials such as this textbook.
+
+Course activities
```

Comment:
What do you think about adding a link to the Booster Rockets section?

Reply:
Added a section on them!